pyproject.toml

```diff
@@ -171,4 +171,4 @@ allow-magic-value-types = ["bytes", "int", "str",]
 max-args = 13
 max-branches = 49
 max-returns = 11
-max-statements = 113
+max-statements = 200
```

Comment:
I would rather bump to the exact value needed for this change. It might help identify refactoring opportunities in future pull requests - but of course not here in a tight loop and without performance testing in CI.